programs/mango-v4/src/instructions/token_force_close_borrows_with_token.rs

```diff
@@ -0,0 +1,202 @@
+use crate::accounts_ix::*;
+use crate::error::*;
+use crate::health::*;
+use crate::logs::TokenBalanceLog;
+use crate::state::*;
+use anchor_lang::prelude::*;
+use fixed::types::I80F48;
+
+pub fn token_force_close_borrows_with_token(
+    ctx: Context<TokenForceCloseBorrowsWithToken>,
+    // which asset tokens are allowed, is checked at # 3
+    asset_token_index: TokenIndex,
+    // token's force_close flag is checked at # 2
```

Comment:
minor, can you use `#2` and `#3`, so it's the same string as below? I put a space there only so github doesn't make it a link to issue # 2